pkg/resolution/common/errors.go

```diff
@@ -51,77 +51,82 @@ func NewError(reason string, err error) *Error {
 }
 
 var (
-	// ErrorRequestInProgress is a sentinel value to indicate that
+	// ErrRequestInProgress is a sentinel value to indicate that
 	// a resource request is still in progress.
-	ErrorRequestInProgress = NewError("RequestInProgress", errors.New("Resource request is still in-progress"))
+	ErrRequestInProgress = NewError("RequestInProgress", errors.New("Resource request is still in-progress"))
+
+	// ErrorRequestInProgress is an alias to ErrRequestInProgress
+	//
+	// Deprecated: use ErrRequestInProgress instead.
+	ErrorRequestInProgress = ErrRequestInProgress
 )
 
-// ErrorInvalidResourceKey indicates that a string key given to the
+// InvalidResourceKeyError indicates that a string key given to the
 // Reconcile function does not match the expected "name" or "namespace/name"
 // format.
-type ErrorInvalidResourceKey struct {
+type InvalidResourceKeyError struct {
```

Comment:
Looking good! Just need to make a similar type alias change for this file, then we should be all set. 👍

Reply:
👍